packages/cli/test/scripts/e2e_test_env.ts

```diff
@@ -14,6 +14,7 @@ const {forkConfig} = defineSimTestConfig({
   BELLATRIX_FORK_EPOCH: bellatrixForkEpoch,
   CAPELLA_FORK_EPOCH: capellaForkEpoch,
   runTillEpoch: Infinity,
+  initialNodes: 2,
```

Comment:
i noticed this  was added.  what does that do?

Reply:
It adjust the genesis delay for the e2e notes, without that default is a bit higher value, so tests take more time to wait for capella fork.